bids/layout/tests/test_path_building.py

```diff
@@ -13,6 +13,15 @@ def layout():
     data_dir = join(get_test_data_path(), '7t_trt')
     return BIDSLayout(data_dir)
 
+def test_path_building_pet(layout):
+    """regression test for https://github.com/bids-standard/pybids/issues/1017
+    
+    TODO: remove if https://github.com/bids-standard/bids-examples/issues/394 is fixed
+          as this test will then be covered 
+          by test_path_building_in_raw_scope
```

Comment:
@effigies do you think that will sound clear enough for future us to know what to do?